spanner-2019-10-01.ddl

```diff
@@ -1,16 +1,25 @@
+-- fxa_uid: a 16 byte identifier, randomly generated by the fxa server
+--    usually a UUID, so presuming a formatted form.
+-- fxa_kid: <`mono_num`>-<`client_state`>
+--
+-- - mono_num: a monotonically increasing timestamp or generation number 
+--             padded to 13 digits, provided by the fxa server
+-- - client_state: the first 16 bytes of a SHA256 hash of the user's sync
+--             encryption key.
+
 CREATE TABLE user_collections (
-  fxa_uid STRING(MAX)  NOT NULL,
-  fxa_kid STRING(MAX)  NOT NULL,
+  fxa_uid STRING(36)  NOT NULL,
+  fxa_kid STRING(38)  NOT NULL,
```

Comment:
IIUC, 13 + 1 + (16bytes as base64 = 24 ASCII) == 38

Reply:
it's as hex (so 32 ascii) = STRING(46)

maybe we overallocate slightly on the boundaries? so STRING(40) and STRING(48)?

Reply:
Is alignment going to matter in any way? Should they just all be powers of 2, 64?